### PR TITLE
Layout and accessibility changed

### PR DESCRIPTION
**Pull request by sabari-venkatesh**
1. minor tweaks to css layout added
2. tabbed navigation for better accessibility
3. support for case insensitive search

### CODE REVIEW

File: tab.js [hunk 1]
```diff
@@ -163,90 +162,85 @@
...
-                    ? selectedTab['windowId'] = data.value
-                    : selectedTab['tabId'] = data.value;
-                }
-            });
-            
-            chrome.extension.sendMessage({type: 'switchTab', selectedTab}, function() {});
-            toggleTabList();
+                .forEach(data => {
+                    if (data.name !== 'class') {
+                        selectedTab['tabId'] ?
```
Comment (madhankumar028) [inline tab.js:183]: this part also thought of refactoring. Check whether is it make sense bro.

`selectedTab[selectedTab.tabId ? 'windowId' : 'tabId'] = data.value;`

File: tab.js [hunk 2]
```diff
@@ -6,23 +6,31 @@
...
+(function () {
 
     'use strict';
 
-    const element       = document.createElement('div');
-    const shadow        = element.attachShadow({mode: 'open'}); // shadowdom
-    const inputElement  = document.createElement('input');
-    const tabList       = document.createElement('div');
+    const
+        VK_UP = 38,
```
Comment (madhankumar028) [inline tab.js:14]: VK stands for?

Comment (sabari-venkatesh) [reply tab.js:14]: **Virtual key**

File: tab.js (hunk 1 above)
Comment (sabari-venkatesh) [reply tab.js:183]: makes sense.. power of JS 💪

**Pull request merged**